custom.css

```diff
@@ -0,0 +1,10 @@
+.form-control { margin: 10px 0px 0px 0px; }
```

Comment:
May be worth re-ordering this file to make it a tad easier to skim past logical groupings.

(ofc the file is tiny, so it doesn't really matter if it's unlikely to grow much)

Reply:
Improved in 2a50356.